### PR TITLE
Added 5 new emotes

### PR DESCRIPTION
**Pull request by SebastianKurp**
Saw Ben's new video and took up on his suggestion to add emotes; so here are 5 new ones.

Emotes Added:

**Coggers**
image

**RareParrot**
image

**CrabPls**
image

**OmegaLuL**
image

**WidePeepoPussy**
image

### CODE REVIEW

**Comment by vi241** [comment]
pls resize the cat to 28x28

**Comment by vi241** [comment]
and you removed a lot of emojis

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 1]
```diff
@@ -319,12 +319,6 @@ export const customEmojis = [
     keywords: ["crypto", "doge", "dodgycoin"],
     imageUrl: "/emotes/dodgyCoin.png",
   },
-  {
-    name: "blaze",
-    short_names: ["blaze"],
-    keywords: ["blaze", "yes", "snark"],
-    imageUrl: "/emotes/blaze.png",
-  },
```
Comment (HarrisonMayotte) [inline kofta/src/app/modules/room-chat/EmoteData.ts]: This shouldn't be removed.

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 2]
```diff
@@ -368,94 +362,34 @@ export const customEmojis = [
...
-    name: "SmudgeCat",
-    short_names: ["SMudgeCat"],
-    keywords: ["meme", "cat"],
-    imageUrl: "/emotes/smudgecat.png",
-  },
-  {
-    name: "Stonks",
-    short_names: ["Stonks"],
-    keywords: ["meme", "crypto", "guy"],
-    imageUrl: "/emotes/stonks.png",
```
Comment (HarrisonMayotte) [inline kofta/src/app/modules/room-chat/EmoteData.ts]: These shouldn't be removed either.

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 3]
```diff
@@ -368,94 +362,34 @@ export const customEmojis = [
...
+    name: "WidePeepoPussy",
+    short_names: ["WidePeepoPussy"],
+    keywords: ["WidePeepoPussy", "peepo", "wide"],
+    imageUrl: "/emotes/widepeepoPussy.png",
   },
   {
-    name: "Stinks",
-    short_names: ["Stinks"],
-    keywords: ["meme", "crypto", "guy"],
-    imageUrl: "/emotes/stinks.png",
```
Comment (HarrisonMayotte) [inline kofta/src/app/modules/room-chat/EmoteData.ts]: This shouldn't be removed.

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 4]
```diff
@@ -368,94 +362,34 @@ export const customEmojis = [
...
+    name: "OmegaLuL",
+    short_names: ["OmegaLuL"],
+    keywords: ["OmegaLuL", "LuL", "Omega"],
+    imageUrl: "/emotes/omegaLul.png",
   },
   {
-    name: "IWMD",
-    short_names: ["IWMD"],
-    keywords: ["meme", "dude", "wednesday", "frog"],
-    imageUrl: "/emotes/wednesday.png",
```
Comment (HarrisonMayotte) [inline kofta/src/app/modules/room-chat/EmoteData.ts]: This shouldn't be removed.

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 5]
```diff
@@ -368,94 +362,34 @@ export const customEmojis = [
...
+    name: "RareParrot",
+    short_names: ["RareParrot"],
+    keywords: ["Rare", "Parrot", "party"],
+    imageUrl: "/emotes/rareParrot.gif",
   },
   {
-    name: "sPonGeBoB",
-    short_names: ["sPonGeBoB"],
-    keywords: ["meme", "spongebob"],
-    imageUrl: "/emotes/sPonGeBoB.png",
```
Comment (HarrisonMayotte) [inline kofta/src/app/modules/room-chat/EmoteData.ts]: The two that were removed here shouldn't be removed either.

**Review by HarrisonMayotte — CHANGES_REQUESTED** [review]
I would recommend changing it to, instead of removing the existing emojis and replacing them, adding new lines in the EmoteData.ts and recommit those changes.

File: kofta/src/app/modules/room-chat/EmoteData.ts (hunk 1 above)
Comment (SebastianKurp) [reply kofta/src/app/modules/room-chat/EmoteData.ts]: Must have pulled an outdated branch, will fix

Comment (HarrisonMayotte) [reply kofta/src/app/modules/room-chat/EmoteData.ts]: All good. 👍🏼

**Comment by SebastianKurp** [comment]
@HMayotte Made the fixes!

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 6]
```diff
@@ -391,13 +493,7 @@ export const customEmojis = [
     keywords: ["Cog", "Coggers", "pepe"],
     imageUrl: "/emotes/coggers.gif",
   },
-  {
-    name: "PurpleDogeHouse",
-    short_names: ["PurpleDogeHouse"],
-    keywords: ["purple", "dogehouse", "doge"],
-    imageUrl: "/emotes/purpledogehouse.png",
-  },
-  {
```
Comment (HarrisonMayotte) [inline kofta/src/app/modules/room-chat/EmoteData.ts]: One concern is just this one

**Review by HarrisonMayotte — CHANGES_REQUESTED** [review]
Looks good except it still looks like one is getting cut out. Not sure why but if you could just pop it back in that'd be great.

**Comment by mmattbtw** [comment]
I would recommend changing the capitalization to the correct capitalization from their [FFZ](https://frankerfacez.com) and [BTTV](https://betterttv.com) emotes.

For Example:
`Coggers` ==> `COGGERS` [BTTV](https://betterttv.com/emotes/5fea6d6304705334f80a9f24)

**Comment by HarrisonMayotte** [comment]
I agree @mmattbtw. Once the changes mentioned above are implemented we'll be good to merge.

**Comment by vi241** [comment]
You removed a lot of emojis, pls add them and then I will merge, your emojis are looking good

File: kofta/src/app/modules/room-chat/EmoteData.ts (hunk 5 above)
Comment (vi241) [reply kofta/src/app/modules/room-chat/EmoteData.ts]: Nothingf from the code should be removed

**Review by vi241 — COMMENTED** [review]
Why change the uppercase, it makes it more confusing

**Review by HarrisonMayotte — APPROVED** [review]

**Comment by vi241** [comment]
will do

**Comment by vi241** [comment]
lets merge and see how will it go

**Pull request merged**